src/util/http/HttpServer.h

```diff
@@ -114,38 +114,38 @@ class HttpServer {
   }
 
   // Get the server port.
-  short unsigned int getPort() const { return _port; }
+  [[nodiscard]] unsigned short getPort() const { return _port; }
```

Comment:
I am not sure, I also used to do this, but this `[[nodiscard]]` doesn't add much value
(if you ignore it, your code will be broken anyway).
You can disable the corresponding clang-tidy check in CLion.
The rule should rather be "every `[[nodiscard]]` should have a reson. I think there is a sonarcloud rule for this, but it doesn't seem to be active.